e2e/pierced-react/functions/_middleware.ts

```diff
@@ -37,6 +37,13 @@ gateway.registerFragment({
 	}),
 });
 
+gateway.registerFragment({
+	fragmentId: 'party-button',
+	prePiercingClassNames: [],
+	routePatterns: ['/__wf/dev.web-fragments.demos.party-button/:_*'],
```

Comment:
May need to be updated when merging and releasing https://github.com/web-fragments/web-fragments/pull/172